prisma/seed.ts

```diff
@@ -34,6 +34,13 @@ async function main() {
       `test family created, test family name is "${testFamily.name}"`,
     );
   }
+
+  const testProject = {
+    name: faker.lorem.word(5),
+    description: faker.lorem.sentence(3),
+  };
+  await prisma.project.create({ data: testProject });
```

Comment:
There should only be one project, so we can check if one already exists and decide not to create another

Reply:
I don't think so, as every time the **seeding** occurs, we **clean up** the DB, so there is no worry about **duplicate records**.